lib/start-controller.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const BaseController = require('./base-controller');
+
+module.exports = class StartController extends BaseController {
+
+  constructor(options) {
+    super(options);
+  }
+
+  getValues(req) {
+    req.sessionModel.reset();
+    super.successHandler.apply(this, arguments);
```

Comment:
Yeah, sorry I hadn't even noticed. Frankly this was a bit of a copy and paste and I wans't paying quite enough attention.